sdk/eventhub/azure-eventhub/tests/livetest/synctests/test_consumer_client.py

```diff
@@ -11,8 +11,8 @@
 @pytest.mark.liveTest
 def test_receive_no_partition(connstr_senders):
     connection_str, senders = connstr_senders
-    senders[0].send(EventData("Test EventData"))
-    senders[1].send(EventData("Test EventData"))
```

Comment:
So previously it looks like we did support sending a single batch, even through it wasn't part of the public API?
I wonder if it might be dangerous to remove the logic that supported this...

Reply:
in our current implement from the public api, a single event is not accepted, an error will be raised:
```
    to_send_batch._load_events(event_data_batch)  # pylint:disable=protected-access
  File "d:\projects\azure-sdk-for-python\sdk\eventhub\azure-eventhub\azure\eventhub\_common.py", line 382, in _load_events
    for event_data in events:
TypeError: 'EventData' object is not iterable
```
so I think it's safe remove the logic.
apart from that, I'd like to keep the implementation compact and simple as I find they're not used. We could always bring the code back when needed.

may I ask what's the concern you have -- do you think that it will break users' usage?